1.0/docs/start/first-element/step-2.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: default
+type: start
+shortname: Start
+title: "Step 2: Add Local DOM"
+subtitle: Build your first Polymer element
+---
+
+<style>
+.caption + pre,.caption + .prettyprint {
+  margin-top: 0px;
+}
+</style>
+
+Next, you'll create a simple element that displays an icon.
+
+In this step, you'll learn about:
+
+*   Creating a custom element using Polymer.
+*   Working with local DOM.
+
+
+<div class="alert alert-info"><strong>Learn more: local DOM.</strong> Local DOM
+lets you add a <em>scoped</em> DOM tree inside an element, with local styles and
+markup that are decoupled from the rest of the web page. The element containing
+the local DOM tree is called a <em>host element</em>. To learn more, see <a
+href="https://www.polymer-project.org/1.0/docs/devguide/local-dom.html">Local
+DOM</a> in the Polymer library docs.
+</div>
+
+## Edit icon-toggle.html
+
+Click on `icon-toggle.html `in the root of the Chrome Dev Editor
+navigation panel. This file contains the skeleton of a custom element.
+
+Unlike most HTML files, this file <em>won't display anything if you load it in a
+browser</em>—it just <em>defines</em> a new element. The demo <em>imports</em>
+`icon-toggle.html` so it can use the `&lt;icon-toggle&gt;`
+element. As you add features to the element in the following steps, they'll show
+up in the demo.
+
+Start by taking a look at the existing code:
+
+
+#### Starting code—HTML imports
+{: .caption }
+
+<pre class="prettyprint">
+&lt;link rel="import"
+  href="../polymer/polymer.html">
+&lt;link rel="import"
+  href="../iron-icon/iron-icon.html">
+</pre>
+
+Key information:
+
+*   The `link rel="import"` element is an <em>HTML import</em>, a new
```

Comment:
I'm trying to reduce or eliminate the blog-style links, but I'll add a link to the article in a Learn More bubble.